src/magicbox/SuggestionsManager.ts

```diff
@@ -186,7 +186,7 @@ export class SuggestionsManager {
       return;
     }
 
-    each(suggestions, (suggestion: Suggestion) => {
+    suggestions.sort((a, b) => b.index || 0 - a.index || 0).forEach(suggestion => {
```

Comment:
Hmm, I think we could simplify the sort function here. If descending order by index, then:
```
suggestions.sort((a,b) => b.index - a.index)
```

Reply:
won't work if an index is undefined

Reply:
I see, I think a test capturing this case would be useful if we do not have one already.

Reply:
In the current test I only added a value to one of the suggestion, that's how I catched that issue 😂